### DISCUSSION TITLE
Remix App as a Chrome Extension

### DISCUSSION THREAD

**Discussion by ethayu** [remix-run/remix#7221] [Q&A] [answered]
Hello, trying to turn my Remix App into a chrome extension. Trying to make it so that the App becomes the default "new tab" page.
Usually this is done by specifying which html file to load, but there aren't any under the project right now. I tried using "public/root.tsx" as a moonshot, but it just displays the code instead of the website. The same occurs for anything that isn't an html file.

Would it be possible to find an index.html file that's usually there in other react projects? Or maybe have npm generate one on build.

**Comment by kiliman** [comment, marked as answer]
Here's an example from none other than @mjackson to statically generate your Remix app. It essentially uses `wget` to crawl your site and save the generated HTML.

https://github.com/mjackson/remix-ssg-example
